src/test/ui/feature-gates/feature-gate-rustdoc_internals.rs

```diff
@@ -0,0 +1,5 @@
+#[doc(keyword = "match")] //~ ERROR: `#[doc(keyword)]` is meant for internal use only
```

Comment:
Why did you add this test? It looks identical to `feature-gate-doc_keyword.rs`. I think it'd be better to just rename the `doc_keyword` test to `rustdoc_internals`.